### PR TITLE
docs: update sample code to replace deprecated <svelte:component>

### PR DESCRIPTION
**Pull request by nanggo**
### What
This PR updates the sample code in the SvelteKit tutorial for universal load functions. Specifically, the `<svelte:component>` tag, which is deprecated in `runes` mode, is replaced with a recommended alternative.

### Why
Using `<svelte:component>` in the tutorial causes the following warning:

```
svelte:component is deprecated in runes mode — components are dynamic by default (svelte_component_deprecated)
```

The editor's solution already uses the updated code, but the sample code in the tutorial did not reflect this change.

### CODE REVIEW

File: apps/svelte.dev/content/docs/svelte/02-runes/02-$state.md
Comment (dummdidumm) [inline apps/svelte.dev/content/docs/svelte/02-runes/02-$state.md:1]: these changes seem unrelated, could you revert them? (if those changes should be made, they need to happen in the Svelte repo)

Comment (nanggo) [reply apps/svelte.dev/content/docs/svelte/02-runes/02-$state.md:1]: I'm sorry about that. Should I just revert or need to force push at my fork repository?
I think something wrong because 06e0046b9a6f6a41bf5343e2a7e525c5a1d5b677 commits in.

**Review by dummdidumm — APPROVED** [review]
thank you!
